platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -92,7 +92,7 @@ extension PolyOut {
                     size: try calculateFileSize(filePath.path),
                     time: "\(Int(floor(time.timeIntervalSince1970)))",
                     name: name ?? URL(fileURLWithPath: filePath.path).lastPathComponent,
-                    id: idFromPodUrl(url) ?? ""
+                    id: url
```

Comment:
This is strange in naming, but as long as it fixes the issue, good enough 👍. We'll have to later polish out the weirdness.

Reply:
Yeah totally, I think id and url get treated as the same and as something different sometimes. But it solves it for now

Reply:
... and add tests if possible.

Reply:
+1 specially for a bug fix we should confirm the new behavior via some tests

Reply:
Feel free to add them